apps/ideaspace/src/components/modules/EditIdea/EditIdea.js

```diff
@@ -260,19 +247,6 @@ const EditIdea = forwardRef(({ initialIdea, onEditSuccess }, ref) => {
   } else {
     return (
       <>
-        {/* <HeadWapper>
-          <Headline>Dev Ideas</Headline>
-          <StyledRanbow>
```

Comment:
are we planning to bring this back in the future? If not, we should delete this instead of commenting.